packages/ses/README.md

```diff
@@ -144,6 +144,29 @@ still closes over the mutable counter.
 Hardening an object graph makes the surface immutable, but does not guarantee
 that methods are free of side effects.
 
+Hardening an object ensures that an attacker cannot alter the API surface of an
+object.
+Hardening "passable" data *does* ensure that a pair of attackers cannot use
+the data to communicate.
+That is, if two guests have access to the same data, they can't manipulate some
+property of the data to send signals.
+
+So, for example, a `Map` instance does not qualify as passable data because
+two guests sharing a map can use it to send messages, by setting and getting
+entries of the map.
+Maps are a capability to read and write, so can be used to communicate.
```

Comment:
MapStores are Remotable, and so are Passable. The reasons we don't allow Maps to be Remotables are messy, but mainly because they allow storage of non-Passable elements. hardened TypedArrays as Remotables would not have this problem.